src/SFA.DAS.ApprenticeFeedback.Database/SFA.DAS.ApprenticeFeedback.Database.refactorlog

```diff
@@ -28,4 +28,11 @@
     <Property Name="ParentElementType" Value="SqlTable" />
     <Property Name="NewName" Value="Ukprn" />
   </Operation>
+  <Operation Name="Rename Refactor" Key="6a429c40-cd4e-4c1d-8f1b-2d52681d9a7e" ChangeDateTime="04/13/2022 14:41:25">
```

Comment:
I think this file was a knock on from the renaming of Id columns as part of trying to fix the other issue, now we've returned back to just `Id` I think this file should be deleted, can talk through what it's for if it helps